src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -32,6 +32,8 @@
 import games.strategy.sound.SoundPath;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.ai.proAI.ProData;
+import games.strategy.triplea.ai.proAI.util.ProBattleUtils;
```

Comment:
What do you think of the resolution in #1665?

Turns out the only thing I was really using the AI for was a call to the battle calculator. No one had said that it was an issue before so I didn't even look at it.